installer/src/systemd_boot/plan.rs

```diff
@@ -277,12 +277,11 @@ fn run_update(bootctl: &Path, esp: &Path) -> Result<()> {
     let status = Command::new(&bootctl).args(args).status()?;
 
     if !status.success() {
-        return Err(format!(
+        info!(
```

Comment:
I don't think `info` shows up when run by default, and it's more apt for a warning anyways ("bootctl may have failed, but also it may just be saying it doesn't need to write out files because they already exist").

```suggestion
        warn!(
```